homeassistant/components/shelly/switch.py

```diff
@@ -173,11 +174,6 @@ def is_on(self) -> bool:
 
         return self.attribute_value in GAS_VALVE_OPEN_STATES
 
-    @property
-    def icon(self) -> str:
```

Comment:
I removed `icon` property because now we can achieve the same with icon translations.